src/header.tsx

```diff
@@ -164,6 +164,8 @@ type HeaderLayoutProps = {
     sideBySideExtraOnDesktop?: boolean;
     children?: void;
     dataAttributes?: DataAttributes;
+    bleed?: boolean;
+    bleedValue?: number;
```

Comment:
Do we want this to be customizable? I don't think so. Design spec says it should be always 40px (@yceballost ?)

Reply:
Hello, I apologize for the delay and not having noticed this error (I didn't put content below in the playroom and I didn't test it in the storybook on the last commit). I'm working on this problem today and I haven't come up with a reasonable solution for this bleed that doesn't influence the size of the external div/layout as a whole, so I haven't commented on anything. 

About Figma, only the fixed part of 40 px was really requested. The idea was to allow, in addition to the request (by default), configurable in case you wanted to use another distance. However, even by request, it is possible to allow only the fixed 40 px and, if requested in the future, do it dynamically.

Note: if you have an idea of a way to apply this bleed exactly to the point 40 px below the beginning of the extra's content, I appreciate it. It's a context that doesn't seem so trivial to me, because the background includes header + extra and both don't have fixed sizes, so it's hard to predict. The way I had thought of it, using the height property at that point unfortunately has side effects.